operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -1053,57 +1053,57 @@ memcachedExporter:
     tag: v0.6.0
     pullPolicy: IfNotPresent
 
-memcached-chunks:
```

Comment:
you mentioned validating that the old setting is not used - I think that's a good idea and we already have similar validation for the `config` property